openwisp_network_topology/migrations/0002_snapshot.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-08-15 10:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0002_auto_20170719_0052'),
```

Comment:
this migration doesn't exists in the openwisp-users repo, what is it?

Reply:
Oh. I observed this earlier too. Upon running the make migrations command, this is being made. Then I manually changed this to `0001_initial` to make the build run.

Reply:
I think we'll need to edit it to prevent the builds from failing.